Update harness-governance-overview.md (#2488)
harness/developer-hub: docs/continuous-delivery/x-platform-cd-features/advanced/cd-governance/harness-governance-overview.md

See [Use Harness Policy As Code for Feature Flags](/docs/platform/Governance/Policy-as-code/using-harness-policy-engine-for-feature-flags).

#### Connectors

> Policies are evaluated against Harness [Connectors](https://developer.harness.io/docs/category/connectors)

Reviewer: ```suggestion
Policies are evaluated against Harness [Connectors](https://developer.harness.io/docs/category/connectors).
```

Policy Set can be configured to evaluate policies on the save of a connector.

> See [Use Harness Policy As Code for Connectors](https://developer.harness.io/docs/platform/Governance/Policy-as-code/sample-policy-use-case#connector-policy-samples)

Reviewer: ```suggestion
See [Use Harness Policy As Code for Connectors](https://developer.harness.io/docs/platform/Governance/Policy-as-code/sample-policy-use-case#connector-policy-samples).
```

#### Template

:::note
Currently, this feature is behind the feature flag, `CDS_OPA_TEMPLATE_GOVERNANCE`. Contact [Harness Support](mailto:support@harness.io) to enable the feature.

> You can configure policies on templates during the creation of the template. This ensures users follow the best practices set by their organization without the organization managing each template’s creation directly. By setting guard rails at creation time it reduces the number of checks needed at runtime to enforce standards.

Reviewer: ```suggestion
You can configure policies on templates during the creation of the template. This ensures that users follow the best practices set by their organization without the organization managing each template’s creation directly. By setting guardrails at creation time, you can reduce the number of checks needed at runtime to enforce standards.
```

See [Use Harness Policy As Code for Feature Flags](/docs/platform/Governance/Policy-as-code/using-harness-policy-engine-for-feature-flags).

#### Connectors

Policies are evaluated against Harness [Connectors](https://developer.harness.io/docs/category/connectors).

> Policy Set can be configured to evaluate policies on the save of a connector.

Reviewer 2: ```suggestion
You can configure a Policy Set to evaluate policies when you save a container.
```

> See [Use Harness Policy As Code for Connectors](https://developer.harness.io/docs/platform/Governance/Policy-as-code/sample-policy-use-case#connector-policy-samples).

Reviewer 2: ```suggestion
Go to [Use Harness Policy As Code for Connectors](https://developer.harness.io/docs/platform/Governance/Policy-as-code/sample-policy-use-case#connector-policy-samples).
```

#### Template

:::note
Currently, this feature is behind the feature flag, `CDS_OPA_TEMPLATE_GOVERNANCE`. Contact [Harness Support](mailto:support@harness.io) to enable the feature.
:::

> You can configure policies on templates during the creation of the template. This ensures that users follow the best practices set by their organization without the organization managing each template’s creation directly. By setting guardrails at creation time, you can reduce the number of checks needed at runtime to enforce standards.

Reviewer 2: ```suggestion
You can configure template policies during the creation of the template. This ensures that you follow your organization's established best practices without the need to manage each template directly. By setting guardrails at creation time, you can reduce the number of checks required at runtime to enforce standards.
```

> Users can now manage the configuration of Templates at configuration time by running policies on Template Save. During Pipeline Save, users can enforce that a specific template is being used. On Pipeline Run users can ensure that the resolved contents of the pipeline meet execution standards.

Reviewer 2: ```suggestion
You can now manage template configuration by running policies on Template Save. During Pipeline Save, you can enforce the use of a specific template. On Pipeline Run, you can ensure that the resolved contents of the pipeline meet execution standards.
```
